README.md

```diff
@@ -69,18 +78,34 @@ cd maestro
 vagrant up
 ```
 
+On Windows also do the following:
+```bash
+VAGRANT_PREFER_SYSTEM_BIN=1 vagrant ssh -c "chmod 400 ~/.ssh/id_rsa"
```

Comment:
What is the reason for doing this?
Would doing `export VAGRANT_PREFER_SYSTEM_BIN=1` not work?

Reply:
Good suggestion. You could export the variable or add it to your bashrc. I have no reason other than I copy-pasted it that way in my shell. It would remove most of my changes. I will rework and resubmit tomorrow.

Reply:
A reason to keep it this way is that this makes the command a one-liner.

Reply:
That's how it ended up that way. OK, I'll keep it cause its easier.